link/src/main/java/com/stripe/android/link/LinkConfigurationCoordinator.kt

```diff
@@ -1,75 +1,32 @@
-import com.stripe.android.core.injection.UIContext
-import com.stripe.android.core.networking.AnalyticsRequestExecutor
-import com.stripe.android.link.injection.DaggerLinkPaymentLauncherComponent
-import com.stripe.android.link.injection.LinkPaymentLauncherComponent
+import com.stripe.android.link.injection.LinkComponent
 import com.stripe.android.link.ui.inline.UserInput
 import com.stripe.android.model.PaymentMethodCreateParams
-import com.stripe.android.networking.PaymentAnalyticsRequestFactory
-import com.stripe.android.networking.StripeRepository
-import com.stripe.android.payments.core.injection.PRODUCT_USAGE
-import com.stripe.android.uicore.address.AddressRepository
-import kotlinx.coroutines.FlowPreview
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.filterNotNull
-import kotlinx.coroutines.flow.flatMapMerge
 import kotlinx.coroutines.flow.map
 import javax.inject.Inject
-import javax.inject.Named
 import javax.inject.Singleton
-import kotlin.coroutines.CoroutineContext
 
 @Singleton
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class LinkConfigurationCoordinator @Inject internal constructor(
-    context: Context,
-    @Named(PRODUCT_USAGE) private val productUsage: Set<String>,
-    @Named(PUBLISHABLE_KEY) private val publishableKeyProvider: () -> String,
-    @Named(STRIPE_ACCOUNT_ID) private val stripeAccountIdProvider: () -> String?,
-    @Named(ENABLE_LOGGING) private val enableLogging: Boolean,
-    @IOContext ioContext: CoroutineContext,
-    @UIContext uiContext: CoroutineContext,
-    paymentAnalyticsRequestFactory: PaymentAnalyticsRequestFactory,
-    analyticsRequestExecutor: AnalyticsRequestExecutor,
-    stripeRepository: StripeRepository,
-    addressRepository: AddressRepository,
-) : NonFallbackInjectable {
-    private val launcherComponentBuilder = DaggerLinkPaymentLauncherComponent.builder()
-        .context(context)
-        .ioContext(ioContext)
-        .uiContext(uiContext)
-        .analyticsRequestFactory(paymentAnalyticsRequestFactory)
-        .analyticsRequestExecutor(analyticsRequestExecutor)
-        .stripeRepository(stripeRepository)
-        .addressRepository(addressRepository)
-        .enableLogging(enableLogging)
-        .publishableKeyProvider(publishableKeyProvider)
-        .stripeAccountIdProvider(stripeAccountIdProvider)
-        .productUsage(productUsage)
+    private val linkComponentBuilder: LinkComponent.Builder,
+) {
+    private val componentFlow = MutableStateFlow<LinkComponent?>(null)
 
-    private val componentFlow = MutableStateFlow<LinkPaymentLauncherComponent?>(null)
-
-    @OptIn(FlowPreview::class)
-    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     val emailFlow: Flow<String?> = componentFlow
         .filterNotNull()
-        .flatMapMerge { it.linkAccountManager.linkAccount }
-        .map { it?.email }
+        .map { it.configuration.customerEmail }
```

Comment:
I’m worried that this might hold onto an outdated email after the user logs out of Link. Can you revert this to use the `LinkAccountManager`?